lib/pulse/core/PulseCore.Atomic.fsti

```diff
@@ -326,8 +326,9 @@ val witness
     (fact:stable_property pcm)
     (v:Ghost.erased a)
     (pf:squash (forall z. compatible pcm v z ==> fact z))
-: stt_ghost
+: stt_atomic

```

Comment:
This change caught a couple of lurking actions mistakenly marked as ghost steps, although they use witness/recall